### PR TITLE
Update generate-integrations-docs.yml

### PR DESCRIPTION
**Pull request by Ancairon**
- Add a cron schedule
- Checkout the latest tag

### CODE REVIEW

**Comment by Ancairon** [comment]
will fix yamllint

File: .github/workflows/generate-integrations-docs.yml [hunk 1]
```diff
@@ -13,12 +16,17 @@ jobs:
     runs-on: ubuntu-latest
     if: github.repository == 'netdata/go.d.plugin'
     steps:
+      - name: Get the latest go release tag
+        id: go-tag
+        run: |
+          echo "::set-output name=releasetag::$(curl "https://github.com/netdata/go.d.plugin/releases/latest" -s -L -I -o /dev/null -w '%{url_effective}' | grep -m 1 -o '[^/]*$')"
```
Comment (ilyam8) [inline .github/workflows/generate-integrations-docs.yml:22]:
```suggestion
          tag=$(curl "https://github.com/netdata/go.d.plugin/releases/latest" -s -L -I -o /dev/null -w '%{url_effective}' | grep -m 1 -o '[^/]*$')
          echo "releasetag=${tag}" >> $GITHUB_ENV
```

**Comment by Ancairon** [comment]
@ilyam8 ran the action again, works, let's merge

File: .github/workflows/generate-integrations-docs.yml [hunk 2]
```diff
@@ -13,12 +16,18 @@ jobs:
     runs-on: ubuntu-latest
     if: github.repository == 'netdata/go.d.plugin'
     steps:
+      - name: Get the latest go release tag
+        id: go-tag
+        run: |
+          tag=$(curl "https://github.com/netdata/go.d.plugin/releases/latest" -s -L -I -o /dev/null -w '%{url_effective}' | grep -m 1 -o '[^/]*$')
```
Comment (ilyam8) [inline .github/workflows/generate-integrations-docs.yml:22]:
```suggestion
          tag="$(curl "https://github.com/netdata/go.d.plugin/releases/latest" -s -L -I -o /dev/null -w '%{url_effective}' | grep -m 1 -o '[^/]*$')"
```

**Pull request merged**